mlir/lib/Conversion/ArithToAMDGPU/ArithToAMDGPU.cpp

```diff
@@ -449,7 +449,8 @@ LogicalResult
 ScalingExtFRewritePattern::matchAndRewrite(arith::ScalingExtFOp op,
                                            PatternRewriter &rewriter) const {
   Location loc = op.getLoc();
-  constexpr int64_t opWidth = 2;
+  constexpr int64_t opOutWidth = 2;
+  constexpr int64_t opInWidth = 8;
```

Comment:
`opInWidth` should be different for different input types right?

- vector of f8E5M2 f8E4M3FN type values of length 1/2/3/4
- vector of f4E2M1FN type values of length 1/2/3/4/5/6/7/8

Reply:
... Right, yeah, `opInWidth` is data type dependent, good catch